ui/src/app/components/common-env-conf-panels/start-stop-environment-button.tsx

```diff
@@ -174,33 +131,13 @@ export const StartStopEnvironmentButton = ({
         borderRadius: '5px 0 0 5px',
       }}
     >
-      {/* TooltipTrigger inside the conditionals because it doesn't handle fragments well. */}
```

Comment:
the conditional related to whether `onClick()` existed or not, and therefore whether to use a `Clickable` - but Clickable works OK without one, so we don't need to do something different.

Reply:
Nice simplification